extensions-core/s3-extensions/src/main/java/org/apache/druid/data/input/s3/S3InputSourceConfig.java

```diff
@@ -63,6 +73,7 @@ public PasswordProvider getSecretAccessKey()
     return secretAccessKey;
   }
 
+
```

Comment:
Please remove this useless blank line.

Reply:
Thanks, but I said this is just a "draft", I'd like to initiate a discussion and see if this feature could be supported.